app-backup/optar/optar-1.0.0.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Codec for encoding data on paper"
+HOMEPAGE="https://github.com/colindean/optar"
+SRC_URI="https://github.com/colindean/${PN}/archive/${PV}-colindean.tar.gz -> ${P}.tar.gz"
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+pgm2ps"
+
+S="${WORKDIR}/${P}-colindean"
+
+DEPEND="media-libs/libpng:*"
+RDEPEND="${DEPEND}
+	pgm2ps? ( media-gfx/imagemagick )"
+
```

Comment:
At a first glance, I see that this package overrides `LDFLAGS` and `CFLAGS`, so you need to override them back to add user's values. As @dwfreed mentioned, you'd also need to override `VERSION` since git is not going to work inside the tarball. Furthermore, the rule for `unoptar` doesn't respect `LDFLAGS`. Generally, you want to write a big patch to sanitize the Makefile and submit it upstream.